src/fork_gc.c

```diff
@@ -744,7 +749,7 @@ static FGCError FGC_parentHandleNumeric(ForkGC *gc) {
 
 static FGCError FGC_parentHandleTags(ForkGC *gc) {
   size_t fieldNameLen;
-  char *fieldName;
+  char *fieldName = NULL;
```

Comment:
Is this where we had the invalid free? https://github.com/RediSearch/RediSearch/pull/7423/files#diff-fdfd94118f1dfc4f48a70f8e1cf6dc2174d164a47054794e6076e89e146ffc22R845